analyzer/javacg-opal/pom.xml

```diff
@@ -45,6 +45,11 @@
             <version>3.1.4</version>
             <type>pom</type>
         </dependency>
+        <dependency>
+            <groupId>org.skyscreamer</groupId>
+            <artifactId>jsonassert</artifactId>
+            <version>1.5.0</version>
```

Comment:
I would suggest to add `<scope>test</scope>` for `jsonassert` dependency.

Reply:
Thanks for your comment I addressed it.